docs/pages/setup/guides/fluentd.mdx

```diff
@@ -224,7 +248,28 @@ and `auth.cas` (certificate, private key, and CA certs, respectively).
 $ tctl auth sign --user=teleport-event-handler --out=identity
 ```
 
-The above sequence should result in one PEM-encoded file: `terraform-identity`.
+The above sequence should result in one PEM-encoded file, `identity`.
+
+</TabItem>
+<TabItem label="Helm Chart" scope={["cloud"]}>
+
+If you are planning to use the Helm Chart, you'll need to generate the keys
+with the `file` format, then create a secret in Kubernetes.
+
+Create the identity using the following command:
+
+```code
+$ tctl auth sign --format=file --user=teleport-event-handler --out=identity
+```
+
+Then create the Kubernetes secret:
```

Comment:
Is there a way to prevent duplicating the `kubectl create secret` command between this tab and the "Configure the plugin" tab above? Maybe remove the command in the "Configure the plugin" tab since we need the `tctl auth sign` command to generate the credentials used in the `kubectl create secret` command?

Reply:
The "Configure the plugin" one is creating a different secret for the TLS credentials generated by `teleport-plugin-event-handler configure`. Or do you mean we should put them into one secret? I'd rather keep them separate to indicate that one's used for connecting to Teleport, the other is used with Fluentd.

Reply:
Oh, I see now. One issue is that in the "Configure the plugin" section, we mention the `kubectl create secret` command before we explain that the `configure` command creates the TLS credential files.

We could add a quick sentence before "To create a Kubernetes secret" explaining why we're creating the secret and how we got the files we're referring to in the `kubectl create secret` command.

Reply:
Good point, thanks!

Reply:
I rephrased the first secret creation, what do you think?